source/gui/__init__.py

```diff
@@ -524,8 +514,17 @@ def __init__(self, frame: MainFrame):
 
 		self.Bind(wx.adv.EVT_TASKBAR_LEFT_DOWN, self.onActivate)
 		self.Bind(wx.adv.EVT_TASKBAR_RIGHT_DOWN, self.onActivate)
-
+	
+	def evaluateUpdatePendingUpdateMenuItemCommand(self):

```

Comment:
If this is something we don't want or expect add-on authors to use, it should be prefixed with an underscore

```suggestion
	def _evaluateUpdatePendingUpdateMenuItemCommand(self):
```

Reply:
Otherwise, we should use deprecations and retain the alias. The current policy is to retain aliases where possible.

Reply:
@seanbudd, it's not very clear to me. Let me rephrase the issues:

There are two `evaluateUpdatePendingUpdateMenuItemCommand` functions (actually, it's a move):
* `gui.MainFrame.evaluateUpdatePendingUpdateMenuItemCommand` that was in NVDA code base and that I remove
* `gui.MainFrame.sysTrayIcon.evaluateUpdatePendingUpdateMenuItemCommand` that was created in this PR.

Questions:
1. In NVDA code base, I do not need gui.MainFrame.evaluateUpdatePendingUpdateMenuItemCommand anymore. Do you recommend to deprecate it or not; it does not cause any problem to keep it; deprecate it would just be useful if we want to clen up the code in the future.
2. I have created gui.MainFrame.sysTrayIcon.evaluateUpdatePendingUpdateMenuItemCommand. Do you recommend to make it private instead?

Reply:
It's a situation of one or the other.
If this is something we want to remove from the API, as it is not expected to be used by API consumers and/or it is not stable, it should be made private.

Otherwise, it should be deprecated and the alias should remain.
I think in this case, keeping the alias is more appropriate as the code should be stable.

Reply:
These are mutually exclusive approaches based on our API policy.

If you wanted to make `gui.SysTrayIcon.evaluateUpdatePendingUpdateMenuItemCommand` private, you would make it as an API breaking change and not retain `gui.MainFrame.evaluateUpdatePendingUpdateMenuItemCommand` as an alias.

Otherwise, you retain the alias with a deprecation notice.
I suggest making change 1, with a deprecation notice.